assets/src/application/ui/display/loadingNotice/LoadingNotice.tsx

```diff
@@ -4,9 +4,15 @@ import { Spin } from 'antd';
 import { _x, sprintf } from '@wordpress/i18n';
 
 import { LoadingNoticeProps } from './types';
+import './style.scss';
 
-const LoadingNotice: React.FC<LoadingNoticeProps> = ({ children, loading, size, ...props }) => {
-	const className = classNames(props.className, 'ee-loading-notice', `ee-loading-notice-${size}`);
+const LoadingNotice: React.FC<LoadingNoticeProps> = ({ children, loading, size = 'large', ...props }) => {
+	const className = classNames({
+		[props.className]: props.className !== undefined,
+		'ee-loading-notice': true,
```

Comment:
since this is true, it can be placed before the object

Reply:
oh i know I just prefer putting everything in the one object because I hate how prettier formats things the other way